### ISSUE TITLE
VTE version not correctly detected on F33

### ISSUE THREAD

**Issue by ValdikSS** [asbru-cm/asbru-cm#749]
**Describe the bug**
Cluster functionality stopped working in v6.2.2.
When the cluster is created with right click on connection group → execute in New Cluster, the commands entered in a single window are not duplicated to other windows.
Console logs the following error:
`*** Vte::Terminal::feed_child: passed too many parameters (expected 2, got 3); ignoring excess at /usr/share/asbru-cm/lib/PACUtils.pm line 3744.`

**To Reproduce**
1. Right-click on connection group → execute in new cluster
2. Wait until all servers are connected, type anything in the console
3. Nothing happens in another cluster windows

**Expected behavior**
The input typed in a single window duplicates into another windows.

**Environment (please complete the following information):**
 - OS: Fedora 33
 - Ásbrú Version: v6.2.2, Asbru repositories.

**Comment by ValdikSS** [comment]
```
$ asbru-cm --verbose
INFO: Ásbrú Connection Manager 6.2.2 (asbru-cm) started with PID 487369
INFO: Desktop environment detected : other
INFO: Config directory is '/home/valdikss/.config/asbru'
INFO: Used config file '/home/valdikss/.config/asbru/asbru.nfreeze'
INFO: Virtual terminal emulator (VTE) version is 0.62
       - has_bright = 1
       - major_version = 0
       - minor_version = 62
       - vte_feed_binary = 1
       - vte_feed_child = 0
INFO: Theme directory is '/usr/share/asbru-cm/res/themes/default'
INFO: Using Gnome tray icon
DEBUG: Starting window position = (maximized, maximized} ; window size (maximized, maximized)
```

Forcing `vte_feed_child=1` fixes the issue.

https://github.com/asbru-cm/asbru-cm/blob/a904aef0ffdb1580fe8dd85019c260f2da1fb223/lib/PACMain.pm#L4743-L4749

Calling `$vte->feed_child('abc', 3);` does not throw an exception, but prints warning into STDERR in my vte library installation.

**Comment by gfrenoy** [comment]
Can you check if #723 is helping ?

**Comment by ValdikSS** [comment]
@gfrenoy, it does. With patch, the program works correctly.

**Comment by gfrenoy** [comment]
As this is fixed in 6.3.0, I'd like to close this one for now and focus on the release of 6.3.0.  Thanks.
